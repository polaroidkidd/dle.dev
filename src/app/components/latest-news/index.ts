export { LatestNews } from "./latest-news"