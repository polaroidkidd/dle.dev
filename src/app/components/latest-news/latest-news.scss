@import "../../styles/main";

.latest-news {
  align-items: center;
  display: flex;
  height: 100%;
  justify-content: center;
  width: 100%;
}