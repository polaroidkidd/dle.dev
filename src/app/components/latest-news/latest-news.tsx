import React from "react";
import "./latest-news"
import Loader from "react-loader-spinner";

const LatestNews = () => {
  return (
    <section className="latest-news">
      {/*<h3 className="heading-tertiary">Latest Blog Entry:</h3>*/}


    </section>
  );
};

export { LatestNews };