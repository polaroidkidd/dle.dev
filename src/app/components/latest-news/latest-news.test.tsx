import React from "react";
import renderer from "react-test-renderer";
import { LatestNews } from "./latest-news";


test("Render Navigation", () => {
  const tree = renderer.create(
    <LatestNews/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
