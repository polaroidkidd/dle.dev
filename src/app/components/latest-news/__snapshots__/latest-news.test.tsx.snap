// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Navigation 1`] = `
<section
  className="latest-news"
/>
`;
